<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDesti</class>
 <widget class="QDialog" name="DlgDesti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient destination</string>
  </property>
  <property name="windowIcon">
   <iconset resource="synoptux.qrc">
    <normaloff>:/images/Maison.png</normaloff>:/images/Maison.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_desti"/>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget_desti">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_heuresortie">
         <property name="text">
          <string>Exit time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_HeureSortie">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy    HH:mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_modifHeure">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="synoptux.qrc">
           <normaloff>:/images/datetime.png</normaloff>:/images/datetime.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_valider">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Validate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_annuler">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>6</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Delete
this
destination</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_quitter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="synoptux.qrc"/>
 </resources>
 <connections/>
</ui>
